### DISCUSSION TITLE
set resolution, sampler and sheduler at start-up

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/stable-diffusion-webui-forge#863] [Q&A] [answered]
where i can set my  
resolution and sampler and sheduler and steps and CFG at start-up (so i want edit the right *.py)

![grafik](https://github.com/user-attachments/assets/fc101008-8320-48ff-90fd-afc6fc8fde7d)

**Comment by E2GO** [comment, marked as answer]
It's not changed: Settings -> Other -> Defaults -> Apply. And don't forget to Apply Settings and Reload UI.
